<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CAMBROSIAS7403" failures="2" tests="15" name="ezNotify.A01CreateEZNotifyEventTest" time="26.107" errors="0" timestamp="2022-11-29T11:34:28 CST" skipped="6">
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.024"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.081">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.074">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.030"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.079"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="18.421">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.075">
    <failure message="expected [The WebElement &quot;&#039;Activate Existing Event&#039; Button&quot; was clicked] but found [The WebElement &quot;&#039;Activate Existing Event&#039; Button&quot; is not clickable]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [The WebElement "'Activate Existing Event' Button" was clicked] but found [The WebElement "'Activate Existing Event' Button" is not clickable]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at utilityClasses.ReportLoggerMethods.click(ReportLoggerMethods.java:126)
at ezNotify.pageMethods.EZ_ActivateEventPage.openActivateEventPage(EZ_ActivateEventPage.java:39)
at ezNotify.A01CreateEZNotifyEventTest.testcases_ezNotify_CreateEZNotifyEventTest(A01CreateEZNotifyEventTest.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.084">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.029"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="6.861">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.080">
    <skipped/>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.076">
    <failure message="expected [The WebElement &quot;&#039;Activate Existing Event&#039; Button&quot; was clicked] but found [The WebElement &quot;&#039;Activate Existing Event&#039; Button&quot; is not clickable]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [The WebElement "'Activate Existing Event' Button" was clicked] but found [The WebElement "'Activate Existing Event' Button" is not clickable]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at utilityClasses.ReportLoggerMethods.click(ReportLoggerMethods.java:126)
at ezNotify.pageMethods.EZ_ActivateEventPage.openActivateEventPage(EZ_ActivateEventPage.java:39)
at ezNotify.A01CreateEZNotifyEventTest.testcases_ezNotify_CreateEZNotifyEventTest(A01CreateEZNotifyEventTest.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- testcases_ezNotify_CreateEZNotifyEventTest -->
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.143"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.027"/>
  <system-out/>
  <testcase classname="ezNotify.A01CreateEZNotifyEventTest" name="testcases_ezNotify_CreateEZNotifyEventTest" time="0.023"/>
  <system-out/>
</testsuite> <!-- ezNotify.A01CreateEZNotifyEventTest -->
